### PR TITLE
update1

### PR DESCRIPTION
**Pull request by ducnv95**
test squash

### CODE REVIEW

**Comment by vanduc95** [comment]
please squash!

**Comment by ducnv95** [comment]
thanks @vanduc95

File: new [hunk 1]
```diff
@@ -1,4 +1,4 @@
 hello world
 happy new year
 v1
-add new line
```
Comment (vanduc95) [inline new]: LGTM

File: newFile [hunk 2]
```diff
@@ -1,4 +1,3 @@
-nguyen van duc
 nguyen van duc1
 nguyen van duc
```
Comment (vanduc95) [inline newFile]: ok

File: new [hunk 3]
```diff
@@ -1,4 +1,4 @@
 hello world
 happy new year
 v1
```
Comment (vanduc95) [inline new]: abc

**Review by vanduc95 — COMMENTED** [review]
ok to me

File: new [hunk 4]
```diff
@@ -1,4 +1,4 @@
 hello world
 happy new year
```
Comment (vanduc95) [inline new:2]: abc

**Pull request closed without merging**